src/main/resources/grammar/Jakt.bnf

```diff
@@ -40,6 +40,14 @@ File ::= NL TopLevelDefinitionList? NL
 
 private TopLevelDefinitionList ::= TopLevelDefinition (StatementSeparator TopLevelDefinition)*
 
+
+Attribute ::= BRACKET_OPEN BRACKET_OPEN {AttributeEntry COMMA?} BRACKET_CLOSE BRACKET_CLOSE
+
+AttributeEntry ::= AttributeName (PAREN_OPEN AttributeParameters PAREN_CLOSE)? (EQUALS AttributeName)?
+AttributeName ::= IDENTIFIER | STRING_LITERAL
+AttributeParameters ::= {AttributeParameter COMMA?}
```

Comment:
I've never seen this curly braces syntax, is this sugar for `(...)*` or `(...)+`? If so I'd prefer we stick with the explicit repetition operators, since curly braces are also used for GrammarKit attributes.